react/src/index.js

```diff
@@ -11,7 +11,6 @@ Amplify.configure({
     mandatorySignIn: true,
     region: config.cognito.REGION,
     userPoolId: config.cognito.USER_POOL_ID,
-    identityPoolId: config.cognito.IDENTITY_POOL_ID,
```

Comment:
so we don't need identity pool, right? just want to confirm.

Reply:
Yup,

Though a bit contradict between
https://docs.amplify.aws/lib/auth/start/q/platform/js/#:~:text=//%20REQUIRED%20only%20for%20Federated%20Authentication%20-%20Amazon%20Cognito%20Identity%20Pool%20ID
and 
https://docs.amplify.aws/lib/auth/advanced/q/platform/js/#:~:text=email%2C%20phone_number%20%7D%0A%20%20%20%20identity_id-,//%20Optional,-%2C%20the%20identity%20id

Reply:
Right! Well, Amplify doc is convoluted with explaining how all those play together.  Just want to note take away tip is that; do we need to expose some (typically private) AWS resources to User? If yes, we need Identity Pool. Otherwise, User Pool is good enough. That is the basic concept. Amplify doc is attempting explaining all different possibilities that library is capable of...

I can pointer to this AWS quick KB article:
https://aws.amazon.com/premiumsupport/knowledge-center/cognito-user-pools-identity-pools/

Anyway, good stuff! Let keep it going.